dockerfiles/Dockerfile.apisix-openresty.rpm

```diff
@@ -0,0 +1,62 @@
+ARG image_base="centos"
+ARG image_tag="7"
+
+FROM ${image_base}:${image_tag}
+
+ARG checkout_v="1.19.3.1"
+
+RUN set -x \
+    # install dependency
+    && yum -y install wget tar gcc automake autoconf libtool make curl git which unzip sudo \
+    && wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && rpm -ivh epel-release-latest-7.noarch.rpm \
+    && yum install -y yum-utils readline-dev readline-devel \
+    # install openssl111
+    && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo \
+    && yum install -y openresty-openssl111-devel \
+    # compile apisix-openresty
+    && yum -y install pcre-devel openssl-devel gcc-c++ patch \
+    && cd $(mktemp -d) || exit 1  \
+    && wget https://openresty.org/download/openresty-"${checkout_v}".tar.gz \
```

Comment:
Please update the current script and call it instead of copying & pasting.

Reply:
update, remove unused dependencies and steps